en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-connector-example.md

```diff
@@ -0,0 +1,116 @@
+# Salesforce Rest API Connector Example
+
+The Salesforce REST Connector allows you to work with records in Salesforce, a web-based service that allows organizations to manage contact relationship management (CRM) data. You can use the Salesforce connector to create, query, retrieve, update, and delete records in your organization's Salesforce data. The connector uses the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_what_is_rest_api.htm) to interact with Salesforce.
+
+## What you'll build
+
+This example explains how to use the Salesforce client to connect with the Salesforce instance and perform the 
+following operations:
+
+* Create an Account 
+* Execute a SOQL query to retrieve the Account Name and ID in all the existing Accounts. 
+
+<img src="/assets/img/connectors/Salesforce.png" title="Using Salesforce Rest Connector" width="800" alt="Using Salesforce Rest Connector"/>
+
+## Configure the connector in WSO2 Integration Studio
+1. Follow these steps to set up the ESB Solution Project and the Connector Exporter Project. 
+{!references/connectors/importing-connector-to-integration-studio.md!}
+
+2. Right click on the created ESB Solution Project and select, -> **New** -> **Rest API** to create the REST API. 
+    <img src="/assets/img/connectors/adding-an-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+
+3. Follow these steps to [generate the Access Tokens for Salesforce](../salesforce-connector/sf-access-token-generation.md) and obtain the Client Id, Client Secret, Access Token and Refresh Token. 
+
+4. First, we will create two defined sequences called `create.xml` and  `retrieve.xml` to create an Account and retrieve data. You can go to the source view of the XML configuration file of the API and copy the following configuration. Replace the `clientSecret`, `clientId`, `accessToken`, `refreshToken` with obtained values in step 3. 
+
+create.xml
```

Comment:
Can we create this inline with the list number? The numbering breaks when we render this currently.

Reply:
Done